docs/import-roster-from-lms.md

```diff
@@ -0,0 +1,78 @@
+## Importing a course roster from your Learning Management System
+
+GitHub Classroom is able to automatically import your course roster from your institution's Learning Management
+System, potentially saving you time and headaches when provisioning a new GitHub Classroom course!
+
+### Prerequisites
+
+This guide assumes that you've successfully [connected GitHub Classroom to you Learning Management System](/help/connect-to-lms). If you haven't, please follow those setup instructions before proceeding to the next steps.
+
+Importing a course roster from your Learning Management System will only work if you have you have enabled GitHub Classroom to receive course roster information per the privacy settings within your Learning Management System.
+
+### Creating a new course roster
+
+To import a new course roster from your Learning Management, select the classroom you would like to import into. Then proceed through the following steps:
+
+   <div class="d-flex flex-justify-around">
+     <img src="/images/help/lms/roster-import/import-roster.gif" class="border" style="width: 75%;">
+   </div>
+
+1. Navigate to classroom settings within GitHub Classroom
+
+2. Select "Roster Management" from the sidebar
+
+3. On the "Add students" page which pops up, click "Import from your Learning Management System"
+
+4. Select how you would like to identify the imported students by:
+
+    - **User ID**: The Student ID as stored in your institution's Learning Management System
+    - **Names**: Student full names
+    - **Emails**: Student emails
+
+    _Note:_ If you do not see the kind of identifier you want, you may have to enable GitHub Classroom to receive student name and email information per your Learning Management System's privacy settings. See [troubleshooting](#troubleshooting) for more advice.
```

Comment:
A little bit of formatting consistency here with notes on other pages.

Also "per", in the sense of "by means of" or "in accordance with", is rarely a good idea (at worst, it's ambiguous; at best, it sounds like stuffy business speak). In this case we can use the ordinary preposition "in". (It's OK to use "per" in the sense of "for each" like "once per day".)

```suggestion
    **Note**: If you do not see the kind of identifier you want, you may have to enable GitHub Classroom to receive student name and email information in your learning management system's privacy settings. See [troubleshooting](#troubleshooting) for more advice.
```